source/data-modeling/field-types.txt

```diff
@@ -0,0 +1,558 @@
+as a ``Hash``. 
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-hash
+   :end-before: # end-hash
+
+Time
+~~~~
+
+You can store values as ``Time`` instances by using the ``Time`` field value.
+``Time`` fields are stored in the time zone configured for your application. To
+learn more about configuring time zones, see the :ref:`Time Zones <time-zones>`
+guide. 
+
+.. TODO: Update Time Zones guide ref if it's changed during standardization
+
+The following example creates a ``Voter`` class and specifies the
+``registered_at`` property as a ``Time`` field:
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-time
+   :end-before: # end-time
+
+.. note::
+
+   Storing a ``Date`` or ``DateTime`` value in a field specified as ``Time``
+   converts the value to ``Time`` when assigned. If you store a string in a
+   ``Time`` field, {+odm+} parses the string by using the ``Time.parse`` method.
+
+Date
+~~~~
+
+You can store the following value types in a field specified as a ``Date``:
+
+- ``Date``: Stores the value as provided.
+- ``Time``: Stores the date portion of the value in the value's time zone.
+- ``DateTime``: Stores the date portion of the value in the value's time zone.
+- ``ActiveSupport::TimeWithZone``: Stores the date portion of the value in the
+  value's time zone.
+- ``String``: Stores the date specified in the string.
+- ``Integer``: Takes the value as if it is a UTC timestamp and
+  converts it into your application's configured time zone. {+odm+} then stores
+  the date taken from that timestamp.
+- ``Float``: Takes the value as if it is a UTC timestamp and
+  converts it into your application's configured time zone. {+odm+} then stores
+  the date taken from that timestamp.
+
+Because converting a ``Time`` or ``DateTime`` discards the time portion, we
+recommend explicitly converting ``String``, ``Time``, and ``DateTime``, objects
+to ``Date`` before assigning them to the field.
+
+.. note::
+
+   When a database contains a string value for a ``Date`` field, the driver
+   parses the value by using the ``Time.parse`` method, then discards the time
+   portion. ``Time.parse`` considers values without time zones to be in local
+   time.
+
+DateTime
+~~~~~~~~
+
+When you assign a value to, or query on a ``DateTime`` field, {+odm+} converts
+the value to a UTC ``Time`` value before sending it to the MongoDB server.
+{+odm+} saves the value with the time zone embedded in the ``DateTime`` object. When
+you retrieve the value, {+odm+} converts the UTC time to the time zone
+configured for your application.
+
+The following example creates a ``Ticket`` class and specifies the ``opened_at``
+field as a ``DateTime`` field:
+
+.. literalinclude:: /includes/data-modeling/field-types.rb
+   :language: ruby
+   :start-after: # start-datetime
+   :end-before: # end-datetime
```

Comment:
S: consider replacing with a more intuitive value or continue using the `Voter` class